daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -894,7 +899,7 @@ object Main {
               case Some(file) => new FileOutputStream(file)
             }
             val infosetType = parseOpts.infosetType.toOption.get
-            val eitherOutputterOrHandler = getInfosetOutputter(infosetType, output)
+            val eitherOutputterOrHandler = getInfosetOutputter(infosetType, output, parseOpts.schema.toOption)
```

Comment:
Is there any value to have a unique infoset kind value for exi schema aware, for example `-I exi` vs `-I exi-sa`? This would allow us to create both and test for things like size and speed.

Reply:
That wouldn't be difficult to implement, so I don't see why not.  As you said it would also make for easier testing.